src/pages/Team.tsx

```diff
@@ -4,37 +4,30 @@ import { BOARD } from "../data";
   return (
-    <div style={{ paddingTop: "10vh", backgroundColor: "white" }}>
-      <SectionTitle header="Board" subheader="Meet the team overseeing the general growth of YCS" />
+    <div style={{ paddingTop: "10vh", paddingLeft:"15vw", paddingRight:"15vw", color: "white" }}>
+      <SectionTitle header="Our Team" subheader="Meet the people that make up y/cs." />
+        <h1 style={{fontWeight: 700, fontSize: "35px", paddingBottom: "3vh"}}>Board</h1>
       <div className="card-grid">
-        {BOARD.slice(0, 7).map((board_member, i) => (
+        {BOARD.slice(0, 12).map((board_member, i) => (
           <CardWrapper key={i}>
-            <img src={board_member.image} alt={board_member.name} loading="lazy" style={{ width: "100%", height: "100%", objectFit: "cover", maxWidth: "200px", maxHeight: "200px" }} />
+            <img src={board_member.image} alt={board_member.name} loading="lazy" style={{ width: "100%", height: "100%", objectFit: "cover", maxWidth: "200px", maxHeight: "200px", borderRadius: "40px" }} />
             <strong style={{ margin: "1rem 0" }}>{board_member.name}</strong>
-            <div>
-              {board_member.role.map((role, i) => (
-                <div style={{ backgroundColor: "lightgrey", color: "#1f232b", padding: "2.5px 5px 2.5px 5px", borderRadius: "5px" }} key={i}>
-                  {role}
-                </div>
-              ))}
-            </div>
+              <div style={{color: "#389C8E", fontFamily: "Space Mono"}} key={i}>
+                  {board_member.role[0].toUpperCase()}
+              </div>
           </CardWrapper>
         ))}
       </div>
-      <SectionTitle header="Project Leads" subheader="Meet the people taking our projects to greater heights" />
+        <h1 style={{fontWeight: 700, fontSize: "35px", paddingTop: "10vh", paddingBottom: "3vh"}}>Product Leads</h1>
       <div className="card-grid">
-        {BOARD.slice(7).map((board_member, i) => (
-          <CardWrapper key={i}>
-            <img src={board_member.image} alt={board_member.name} loading="lazy" style={{ width: "100%", height: "100%", objectFit: "cover", maxWidth: "200px", maxHeight: "200px" }} />
-            <strong style={{ margin: "1rem 0" }}>{board_member.name}</strong>
-            <div>
-              {board_member.role.map((role, i) => (
-                <div style={{ backgroundColor: "lightgrey", color: "#1f232b", padding: "2.5px 5px 2.5px 5px", borderRadius: "5px" }} key={i}>
-                  {role}
+        {BOARD.slice(12).map((board_member, i) => (
+            <CardWrapper key={i}>
+                <img src={board_member.image} alt={board_member.name} loading="lazy" style={{ width: "100%", height: "100%", objectFit: "cover", maxWidth: "200px", maxHeight: "200px", borderRadius: "40px" }} />
+                <strong style={{ margin: "1rem 0" }}>{board_member.name}</strong>
+                <div style={{color: "#389C8E", padding: "2.5px 5px 2.5px 5px", borderRadius: "5px" }} key={i}>
```

Comment:
I see that there is precedent for styling these in the JSX, but imo it would be nice in a later PR to move all styling to a stylesheet